c/main.c

```diff
@@ -4,7 +4,9 @@
 #include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
+#if !defined(_WIN32)
 #include <unistd.h>
```

Comment:
Maybe we can remove this header entirely? It doesn't seem to affect the build when I try it.

At the same time, if we're adding Windows support here, we should probably figure out a way to add Windows CI testing in `.github/workflows/ci.yml`.

Reply:
Yes, for current content of main.c, including of this file can be removed. Probably it was left some experimentations that even not was committed.

Agree, adding of Windows CI testing environment would be right step to protect all changes proposed here and prevent regression changing that broke compile able at MSVC.